src/fides/api/ops/task/graph_task.py

```diff
@@ -114,6 +116,20 @@ def result(*args: Any, **kwargs: Any) -> Any:
                     )
                     self.log_skipped(action_type, exc)
                     return default_return
+                except (SkippingConsentPropagation,) as exc:
+                    logger.warning(
+                        "Skipping collection {} for privacy_request: {}",
+                        self.traversal_node.address,
+                        self.resources.request.id,
+                    )
+                    self.log_skipped(action_type, exc)
+                    for pref in self.resources.request.privacy_preferences:
+                        # For consent reporting, also caching the given system as skipped for all historical privacy preferences.
+                        pref.cache_system_status(
+                            self.connector.configuration.system_key,
+                            ExecutionLogStatus.skipped,
```

Comment:
If no requests are fired for a consent saas connector, we add a skipped status for every privacy preference history attached to the privacy request.  If we instead have a scenario where some preferences and relevant and some are not, so we still fire the request, those "skipped" logs are added on the saas_connector.run_request.